pandas/tests/series/indexing/test_setitem.py

```diff
@@ -126,6 +127,16 @@ def test_setitem_boolean_different_order(self, string_series):
 
         tm.assert_series_equal(copy, expected)
 
+    def test_setitem_boolean_nullable_int_types(self, any_nullable_int_dtype):
+        # GH: 26468
+        ser = Series(pd_array([5, 6, 7, 8], dtype="Int64"))
```

Comment:
shouldn't you be using the any_nullable_int_dtype fixture here?

Reply:
Sorry, somehow missed that. Thanks for pointing it out. Changed it